buildah/Dockerfiles/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM centos:7
```

Comment:
Should be based on UBI 8

Reply:
i can't find buildah and podman package on ubi 8  :(